examples/nginx-access-logs-streaming/README.md

```diff
@@ -0,0 +1,640 @@
+# Analyzing Nginx Access Logs using Timeplus Proton 
+In a [recent blog post](https://www.timeplus.com/post/log-stream-analysis), we walked through three different ways of using Timeplus Proton to ingest and analyse a log file in real-time.
+
+I’ll explore a similar theme in the first part of this post: ad-hoc analysis of web traffic, in real-time, using Timeplus Proton. I will use a Node.js blog that is behind an Nginx web server for the analysis. 
+
+In the second part of this post, I will analyze past traffic stats for the blog. I'll compare historical traffic stats obtained using Timeplus Proton with traffic stats reported by [Umami](https://umami.is). (Umami is an open source, privacy-focused alternative to Google Analytics.) 
+
+The numbers from Timeplus Proton should be of higher accuracy than the numbers reported by Umami because Umami, like Google Analytics, is a JavaScript-based analytics product and thus susceptible to under-reporting traffic from users with ad blocking enabled.
+
+## Background
+The blog that we will using is my personal blog which I launched back in 2020. It's a [Ghost](https://ghost.org/) blog hosted on AWS. Ghost is written in JavaScript/Node.js and is completely [open source](https://github.com/tryghost/ghost).
+
+In production, the Ghost blog is deployed behind Nginx which serves as a reverse proxy, so the Ghost blog is in fact two services: 
+* a Node.js web app that listens on port `2368`
+* an Nginx web server that listens for incoming traffic on the standard HTTP/HTTPS ports `80` and `443`. It handles SSL termination for the Node.js app. 
```

Comment:
a Nginx ..

Reply:
Please let me know if you agree with my response [here](https://github.com/timeplus-io/proton/pull/686#discussion_r1591690574).